### PR TITLE
Short Version Test for Greg Morancey

### PR DESCRIPTION
**Pull request by monstersLA**
Completed short test per your request.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,28 +1,44 @@
...
+		$('input[value=Post]').on('click',function(event){ /* Check if form was submitted */
+			var $thoughts = $("#search").find('input').val(); /* Find input field data */
+			var dNow = new Date(); /* Set Date Time */
+			var utc = new Date(dNow.getTime() + dNow.getTimezoneOffset() * 60000)
+			var utcdate= (utc.getMonth()+1) + '/' + utc.getDate() + '/' + utc.getFullYear() + ' ' + utc.getHours() + ':' + utc.getMinutes() + ':' + utc.getSeconds();
+			var $day = 	utc.getSeconds();  /* Find seconds */
+			var $answer = ($thoughts+' - '+$day+' <br />');  /* Combine & break data  */
+		$('#page').prepend($answer);  /* Add last input to top of list */
+		return false;  /* Allow the browser to place multiple inputs */
+		document.write($answer);  /* Print to Div */
```
Comment (LorenzoGangi) [inline index.html]: Im not sure you need this document.write. You already added the $answer to the DOM on line 17. What are you trying to do here?

File: index.html [hunk 2]
```diff
@@ -1,28 +1,44 @@
...
+	<link rel="stylesheet" type="text/css" href="styles.css" />
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+	<script>
+		$(document).ready(function(){
+		$('input[value=Post]').on('click',function(event){ /* Check if form was submitted */
+			var $thoughts = $("#search").find('input').val(); /* Find input field data */
+			var dNow = new Date(); /* Set Date Time */
+			var utc = new Date(dNow.getTime() + dNow.getTimezoneOffset() * 60000)
+			var utcdate= (utc.getMonth()+1) + '/' + utc.getDate() + '/' + utc.getFullYear() + ' ' + utc.getHours() + ':' + utc.getMinutes() + ':' + utc.getSeconds();
+			var $day = 	utc.getSeconds();  /* Find seconds */
```
Comment (LorenzoGangi) [inline index.html]: I think a slightly less complicated way of getting the current time's seconds would be
let seconds= new Date().getSeconds();
or
let seconds = dNow.getSeconds();

File: index.html (hunk 1 above)
Comment (monstersLA) [reply index.html]: Yup..meant to remove it. Was writing to div

File: index.html (hunk 2 above)
Comment (LorenzoGangi) [reply index.html]: Your variable name is a little confusing as well. Typically prepending a variable with $ indicates its an jQuery instance. Also it seems like you are assigning it the number of seconds but the variable name is $day.

Comment (monstersLA) [reply index.html]: Much less complicated

Comment (monstersLA) [reply index.html]: Thanks for the review Lorenzo. I appreciate it. Any other critiques?

**Pull request closed without merging**